sqlglot/parser.py

```diff
@@ -327,10 +314,10 @@ def _parse_exists(self, not_=False):
         )
 
     def _parse_create(self):
-        temporary = bool(self._match(TokenType.TEMPORARY))
-        replace = bool(self._match(TokenType.OR) and self._match(TokenType.REPLACE))
+        temporary = self._match(TokenType.TEMPORARY)
+        replace = self._match(TokenType.OR) and self._match(TokenType.REPLACE)
```

Comment:
Side question: do we support `IF NOT EXISTS`?

Reply:
yes, look at identity.sql line 232